js/coinflex.js

```diff
@@ -0,0 +1,2391 @@
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38432",
+        //         "toAccount": "38774",
+        //         "id": "758113170128764931",
+        //         "status": "COMPLETED",
+        //         "transferredAt": "1651428178967"
+        //      }
+        //
+        // transfer
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38422",
+        //         "toAccount": "38776",
+        //         "transferredAt": "1651704762775"
+        //     }
+        //
+        const currencyId = this.safeString (transfer, 'asset');
+        const timestamp = this.safeString (transfer, 'transferredAt');
+        const fromAccount = this.safeString (transfer, 'fromAccount');
+        const toAccount = this.safeString (transfer, 'toAccount');
+        let status = this.parseTransactionStatus (this.safeString (transfer, 'status'));
+        // if case of 'transfer', as status field is not available, 'transferredAt' prop should mean that transfer was done
+        if (status === undefined && timestamp !== undefined) {
+            status = 'ok';
+        }
+        return {
+            'id': this.safeString (transfer, 'id'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'currency': this.safeCurrencyCode (currencyId, currency),
+            'amount': this.safeNumber (transfer, 'quantity'),
+            'fromAccount': fromAccount,
+            'toAccount': toAccount,
+            'status': status,
+            'info': transfer,
+        };
+    }
+
+    async buildOrderRequest (market, type, side, amount, price = undefined, params = {}) {
+        const clientOrderId = this.safeString (params, 'clientOrderId');
+        const maxCOI = '9223372036854775807';
+        if ((clientOrderId !== undefined) && Precise.stringGt (clientOrderId, maxCOI)) {
+            throw new InvalidOrder (this.id + ' createOrder() param clientOrderId should not exceed ' + maxCOI);
+        }
+        const orderType = this.convertOrderType (type);
+        const stopPrice = this.safeNumber (params, 'stopPrice');
+        const limitPrice = this.safeNumber (params, 'limitPrice');
+        const stopPriceIsDefined = stopPrice !== undefined;
+        const orderTypeIsStop = orderType === 'STOP';
+        const isStopOrder = stopPriceIsDefined || orderTypeIsStop;
+        if (orderTypeIsStop) {
+            if (!stopPriceIsDefined) {
+                throw new ArgumentsRequired (this.id + ' createOrder() requires params["stopPrice"] for stop orders');
+            } else if (limitPrice === undefined && price === undefined) {
+                throw new ArgumentsRequired (this.id + ' createOrder() requires "price" argument or params["limitPrice"] as a limit price for stop orders, as stop-market orders are not supported on this exchange');
```

Comment:
Using the UI stop-market orders are possible, are you sure we can't do the same using the API?

![image](https://user-images.githubusercontent.com/43336371/167637071-bc1f852c-9e9e-422a-9b3e-ec0d67ae619a.png)

Reply:
where do you see them? can you link please

Reply:
@ttodua This image is a PrintScreen of CoinFlex's Web UI 🙂 ,  I don't know if they also support this order type (stop market) using the API, I was asking to confirm it

Reply:
that's what I wanted to ask, i've been browsing coinflex UI and couldnt find such page what you had screenshoted. which pair/market it was, so i can check in webui?

Reply:
@ttodua please disregard this message 🤦 🤦‍♂️  and sorry I opened the wrong exchange yesterday 😅

Reply:
ah no problem nvm ;)